apps/client/src/features/control/message/MessageControl.tsx

```diff
@@ -59,6 +61,17 @@ export default function MessageControl() {
           data-testid='toggle timer blackout'
         />
       </div>
+      <div>
+        <InputRow
+          label='External Message'
+          placeholder='-'
+          readonly={true}
+          text={data.externalMessage.text || ''}
+          visible={data.externalMessage.visible || false}
+          changeHandler={(newValue) => setMessage.externalText(newValue)}
+          actionHandler={() => setMessage.externalVisible(!data.externalMessage.visible)}
+        />
+      </div>
```

Comment:
I am not sure if we should be setting this at all here. I thought we agreed that this was only for visibility and we wouldnt have controls in our interface?